pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/MhloLegalizeUtils.cpp

```diff
@@ -25,6 +25,17 @@
 namespace mlir {
 namespace mhlo {
 
+std::vector<int64_t> normalizeDimIndex(ArrayRef<int64_t> dims, int64_t rank) {
```

Comment:
I see this function defined in #416  as well. Should we just remove one?

Reply:
I will rebase it later.